docs/portal/developer-portal/advanced/Enabling_K3s.md

```diff
@@ -0,0 +1,317 @@
+```bash
+$ cray artifacts list user --format json
+{
+  "artifacts": [
+    {
+      "Key": "subuid",
+      "LastModified": "2023-02-21T23:41:43.948000+00:00",
+      "ETag": "\"c543aebb9b40bcf48879885734447090\"",
+      "Size": 145686,
+      "StorageClass": "STANDARD",
+      "Owner": {
+        "DisplayName": "User Service User",
+        "ID": "USER"
+      }
+    },
+    {
+      "Key": "subgid",
+      "LastModified": "2023-02-21T23:41:43.948000+00:00",
+      "ETag": "\"73032ede132e44d2c1bc567246901737\"",
+      "Size": 145686,
+      "StorageClass": "STANDARD",
+      "Owner": {
+        "DisplayName": "User Service User",
+        "ID": "USER"
+      }
+    }
+  ]
+}
+```
+
+### Podman Image
+A container image suitable for users should be available in a container image registry accessible from the UAN.
+
+
+## Configuring with Configuration Framework Service (CFS)
+
+After completing the prerequisites section, the following should be established to proceed with configuring a UAN to run K3s.
+
+- A fully configured UAN
+- An IPAddress start and end range to assign to MetalLB
+- Prepared subuid and subgid files 
+
+### Configuration Files and Playbook
+
+Configuration for K3s and related services are controlled in the following ansible files in the UAN VCS repository:
+```bash
+$ ls uan-config-management/vars/ | egrep "k3s|sshd|helm"
+uan_helm.yml
+uan_k3s.yml
+uan_sshd.yml
+```
+The ansible playbook to install and configure K3s may be found here:
+```bash
+$ ls uan-config-management | grep k3s
+k3s.yml
+```
+
+### Ansible Roles
+
+The following Ansible roles are provided in the `uan-config-management` repository in VCS. There are `README.md` files in each Ansible role directory in this repository that provide further details.
+
+- **uan-k3s-install**: Download and stage K3s assets necessary to initialize and configure k3s on a node
+- **uan-k3s-stage**: Install and configure K3s on a node
+- **uan-helm**: Perform tasks to initialize an environment to install helm charts
+- **uan-haproxy**: Deploy a list of HAProxy charts to the k3s cluster
+- **uan-metallb**: Deploy the MetalLB chart to the k3s cluster
+- **uan-sshd**: Create and enable instances of `sshd` with systemd
+
+### Artifactory Assets
+
+UAN uploads artifacts to deploy to the UAN control-plane node in a new nexus repository:
+
+- uan-2.6.XX-third-party
+
+The following Nexus group repository is created and reference the aforementioned UAN Nexus raw repos.
+
+- uan-2.6-third-party
+
+This repository will contain the installer for K3s, Helm charts for HAProxy and MetalLB, etc.
+
+### Validation Tests 
+
+To validate the K3s cluster once deployed, see the [Validation Checks](#validation-checks) section of this document for details.
+
+## Getting Started
+
+Perform the following tasks to get it going! todo fix this
+
+### MetalLB 
```

Comment:
This and the following Level 3 sections should be converted into a single ordered list since they depict a sequence of steps (and their explanations and examples), similar to what I did above with the prerequisites.

Reply:
I'm hesitant to do this as this is not a 100% prescriptive step-by-step set of instructions. Would it be acceptable to start with a note that each of the sections should be read and understood to determine how a site may want to configure each of the components?

Reply:
There are some optional steps, supporting info, and substeps. I think it would still be better to refactor this as a procedure. What if I do the change and use this web GUI to suggest the change? Then you can accept/reject it.

Reply:
Since this is a Technical Preview feature, we can leave it as it is in this commit, with the explanatory paragraph at the top saying that these sections need to be read in full before being performed. I'll procedure-fy this later.